src/components/Header/index.js

```diff
@@ -1,6 +1,7 @@
 import { useIsAuthenticated } from '@azure/msal-react';
 import { AppBar, Button, IconButton, Tab, Tabs, Toolbar, Typography } from '@material-ui/core';
 import EventIcon from '@material-ui/icons/Event';
+import SportsFootballIcon from '@material-ui/icons/SportsFootball';
```

Comment:
I've uploaded a set of icons that MUI provides for sports, perhaps it would be a good idea to bring it up to our customer and ask if they would like a completely new icon or if any of the sport icons would suit their taste.